src/theme/theme.ts

```diff
@@ -277,6 +280,13 @@ const theme = createTheme({
       fontSize: "80px",
       lineHeight: "96px",
     },
+    h2: {
```

Comment:
I added an `h2` but I'm not sure this is what Design intended, it matched up with the font choice being used in the DSP Pricing Plan. 

However, there is a large conversation with Design that I want to start as the font's probably should consider accessibility and currently in the design it jumps around between `h2` to `h4` without consideration of hierarchy, only because the `fontFamily` and `fontSize` matches what is shown in Figma. My question would also be: can we get labeled style names in Figma so there isn't so much variation between text styles and we can match our theme template in a more consistent manner.